apps/web/app/api/pipeline/route.ts

```diff
@@ -163,14 +165,16 @@ export async function POST(request: Request) {
           },
         });
         if (responseCount === survey.autoComplete) {
-          await prisma.survey.update({
+          const updatedSurvey = await prisma.survey.update({
```

Comment:
Can we also use the `updateSurvey` method from services here, then we don't have cache revalidation issues ;-)